server/eb/app.py

```diff
@@ -0,0 +1,53 @@
+"""cellxgene AWS elastic beanstalk application"""
+
+import sys
+import os
+import logging
+
+logging.basicConfig(
+    filename="/opt/python/log/app.log",
+    level=logging.DEBUG,
+    format="%(asctime)s.%(msecs)03d %(levelname)s %(module)s - %(funcName)s: %(message)s",
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
+logging.getLogger().addHandler(logging.StreamHandler(sys.stdout))
+
+SERVERDIR = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(SERVERDIR)
+
+try:
+    from server.common.app_config import AppConfig
+    from server.app.app import Server
+    from server.data_common.matrix_loader import MatrixDataCacheManager
+except Exception:
+    logging.exception("Exception importing server modules")
+    sys.exit(1)
+
+try:
+    dataroot = os.getenv("CXG_DATAROOT")
+    if dataroot is None:
+        logging.error("CXG_DATAROOT environment variable must be set")
+        sys.exit(1)
+
+    app_config = AppConfig(dataroot=dataroot)
```

Comment:
For this interim PR (ie, prior to having a more robust configuration mechanism), I would add explicit setting of any configuration where we must have a certain value and the default might change.   Specifically, I would explicitly disable annotations.

Reply:
Yes agreed.   I'll fill in all the config parameters.
annotations are already turned off a few lines down.
What about diffexp? Should that be enabled here or disabled?

Reply:
for now, I think it is fine to enable diffexp.   When we do a more robust configuration mechanism, it can be refined.